frontend/packages/portal-frontend/src/compound/utils.ts

```diff
@@ -69,3 +72,27 @@ export function mapEntrezIdToSymbols(
 
   return geneSymbols;
 }
+
+function isMatrixDataset(d: unknown): d is MatrixDataset {
+  return (
+    typeof d === "object" &&
+    d !== null &&
+    "units" in d &&
+    typeof (d as any).units === "string"
+  );
+}
+
+export async function getHighestPriorityCorrelationDatasetForEntity(
+  compoundID: string
+) {
+  const datasets = await cached(breadboxAPI).getDatasets({
+    feature_id: compoundID,
+    feature_type: "compound_v2",
+  });
+
+  const match = datasets.find(
+    (d) => isMatrixDataset(d) && d.priority === 1 && d.units === "log2(AUC)"
+  );
```

Comment:
There are 2 OncRef datasets with a priority of 1. I wasn't sure how to identify the proper OncRef correlations dataset (we wanted logged units) without hard coding the units. I don't know if I can assume that all future correlation datasets will be "log2(AUC)", so this might not work long term. 

Thoughts?